README.md

```diff
@@ -9,51 +9,35 @@ with a fully functional system straight from your database schema. If
 you have more specialized requirements, HatchifyJS makes it easy to
 customize every part of the application to meet your needs.
 
-HatchifyJS enables you to make changes to your database schema and
-customize app behavior independently. When using code generation tools
-you have to write your schema and then generate your code, but once you
-start making customizations you cant re-run the generator without losing
-your customizations.
-
-HatchifyJS is NOT code generation, it's a system of modular and
-hierarchical libraries that can be consumed piecemeal to use as much or
-as little of HatchifyJS abilities as you require.
-
-- [About HatchifyJS](#GettingStartedHatchify-AboutHatchifyJS)
-- [Project Setup](#GettingStartedHatchify-ProjectSetup)
-- [Schemas](#GettingStartedHatchify-Schemas)
-  - [Model
-    Relationships](#GettingStartedHatchify-ModelRelationships)
-- [Backend - The Hatchify
-  Middleware](#GettingStartedHatchify-Backend-TheHatchifyMiddleware)
-  - [Using the server
-    endpoints](#GettingStartedHatchify-Usingtheserverendpoints)
-    - [Creating a
-      resource](#GettingStartedHatchify-Creatingaresource)
-    - [Listing
-      resources](#GettingStartedHatchify-Listingresources)
-    - [Fetching a single
-      resource](#GettingStartedHatchify-Fetchingasingleresource)
-    - [Updating a
-      resource](#GettingStartedHatchify-Updatingaresource)
-    - [Deleting a
-      resource](#GettingStartedHatchify-Deletingaresource)
-  - [Seeding data](#GettingStartedHatchify-Seedingdata)
-- [Frontend with React &
-  MUI](#GettingStartedHatchify-FrontendwithReact&MUI)
-  - [Rendering a List](#GettingStartedHatchify-RenderingaList)
-- [Next Steps](#GettingStartedHatchify-nextSteps)
-  - [Rendering an empty list](#GettingStartedHatchify-renderingAnEmptyList)
-- [Need help or have
-  questions?](#GettingStartedHatchify-Needhelporhavequestions?)
+Unlike code generation tools—which allow you to write your schema and then generate your code, but once you start making customizations you can't re-run the generator without losing your customizations—HatchifyJS enables you to make changes to your database schema and customize app behavior **independently**. This is because HatchifyJS is **not** code generation—it's a system of modular and hierarchical libraries that can be consumed piecemeal to use as much or as little of HatchifyJS abilities as you require.
```

Comment:
This paragraph consolidates the previous two, and I think it reads a little easier